pkgs/development/python-modules/pyutilib/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "PyUtilib";
+  name = "${pname}-${version}";
```

Comment:
no `name`

Reply:
What do you mean?

Reply:
You can leave the line out because `name` is automatically set to `${pname}-${version}` by `buildPython*`.